### ISSUE TITLE
md-tooltip does not work together with md-menu

### ISSUE THREAD

**Issue by mathiasmoeller** [angular/material#6662]
I am using a md-menu with a lot of tooltips inside the menu element.
In material version 1.0.0-rc4 it was working fine. Since upgrading to 1.0.1 it is broken.
The tooltips all appear when hovering the button to open the md-menu, but when it is open, no tooltips are shown.
See the demo:
http://one-logic.github.io/md-color-menu/
(after opening the menu every color should have a tooltip)

this is the code:
```html
     <md-menu md-position-mode="target-right target">
        <div ng-click="vm.openMenu($mdOpenMenu, $event)" ng-transclude=""></div>
          <md-menu-content class="md-cm">
            <div></div>
            <div class="md-cm-swatches" layout="row">
              <div ng-repeat="swatch in vm.colors" layout=column>
                <div ng-repeat="color in swatch" class="md-cm-color" ng-style="color.style" ng-click="vm.selectColor(color)" layout="row" layout-align="center center">
                  <span ng-if="color.name == vm.color.name">&#10004;</span>
                  <md-tooltip>{{color.name}}</md-tooltip>
                </div>
              </div>
            </div>
          </md-menu-content>
        </md-menu>
````

**Comment by maelfrize** [comment]
+1
Same issue here.

**Comment by StritziK** [comment]
Same problem with me, glad I'm not alone

**Comment by yigaldviri** [comment]
+1

**Comment by stevezau** [comment]
+1

**Comment by STOpandthink** [comment]
+1, here is the codepen: http://codepen.io/anon/pen/wGveJW
I noticed that the issue happens only if there is an "ng-if" in the mix.

**Comment by mathiasmoeller** [comment]
@STOpandthink i have the same unwanted behavior when using `ng-show` instead of `ng-if`

**Comment by yulin-dong** [comment]
+1

**Comment by darwin-gautalius** [comment]
after looking at the code, I try giving `style="pointer-events: auto"` to `md-tooltip`'s parent as a workaround. and it works perfectly.
```html
<any style="pointer-events: auto">
  <md-tooltip>foo</md-tooltip>
  ...
</any>
```

**Comment by pkonya** [comment]
+1 same problem, but when i put ng-show instead of nf-if the behaviour of tooltip is normal.

**Comment by waterplea** [comment]
Why was it closed? The issue still present in 1.1.0.

**Comment by ThomasBurleson** [comment]
We will not fix this for ngM1.x.
